packages/material-ui/src/styles/createTypography.d.ts

```diff
@@ -28,6 +28,28 @@ export interface FontStyle
     htmlFontSize: number;
   }> {}
 
+type NormalCssProperties = CSS.Properties<number | string>;
```

Comment:
I copied these typings from `@material-ui/styles/withStyles` to avoid dependency. We couldn't use here `CSSObject` from `experimentalStyled.d.ts` as the utilities in `@material-ui/styles` do not have the fallback (array values) in the typings. Is this intentional? Does JSS not support this?